client/src/components/modals/modify-bet/ModifyBetDetailsForm.tsx

```
@@ -1,20 +1,13 @@
 import "./ModifyBetDetailsForm.css";
 
-import {
-  Dispatch,
-  SetStateAction,
-  SyntheticEvent,
-  useEffect,
-  useState,
-} from "react";
+import { Dispatch, SetStateAction, SyntheticEvent, useEffect } from "react";
 
 import { Button, Error, Loading } from "../../";
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)